content/blog/2019/mesh-federation/index.md

```diff
@@ -0,0 +1,1816 @@
+---
+title: Mesh Federation
+subtitle: Connect multiple service meshes ad hoc, with limited service exposure and strict cross-cluster access control
+description: Connect multiple service meshes ad hoc, with limited service exposure and strict cross-cluster access control.
+publishdate: 2019-08-12
+attribution: Vadim Eisenberg (IBM)
+keywords: [traffic-management,multicluster,security,gateway,tls]
+---
+
+Managing applications across multiple Kubernetes clusters is currently one of the hottest topics in Kubernetes and Istio
+communities. You may need to split your applications between multiple clusters for different reasons, for example:
+
+-  scalability limits of a single cluster
+-  separation of production/dev/test/staging environments
+-  different security and compliance requirements for different parts of the application
+-  running parts of the application on multiple public clouds, and on a private cloud
+
+Currently there are [three patterns](/docs/setup/kubernetes/install/multicluster/) in Istio to create a
+[multicluster service mesh](/docs/concepts/multicluster-deployments/#multicluster-service-mesh).
+A _multicluster service mesh_ is a single _logical_ service mesh, spread among multiple Kubernetes clusters.
+
+Such a service mesh has the following characteristics:
+
+- **uniform identical naming** of namespaces and services. The `withdraw` service in the `accounts` namespace in one
+cluster has the same functionality and API as the `withdraw` service in the `accounts` namespace in all other clusters
+in the mesh.
+- **expose-all by default**. All the services are exposed by default to all the clusters in the mesh.
+- **common identity and common trust** are established between all the clusters in the mesh.
+
+The three patterns differ by the following aspects:
+
+- The clusters are on a **single network** or on **multiple networks**. Single network means that the pods in one
+cluster can reach the pods in other clusters directly. Multiple networks means that the pods in one cluster can reach
+the pods in other clusters in the mesh only through corresponding ingress gateways.
+- The Istio control planes are **shared** between the clusters in the mesh or each cluster has its own **dedicated**
+control plane.
+
+There are use cases when you want to connect service meshes in separate clusters while limiting exposure of services
```

Comment:
Ok so this is mesh federation.  It took way too long to get here. I suggest moving discussion of what mesh federation is the top.

Reply:
@nmittler I agree that we can remove the single multi-cluster meshes descriptions altogether and start with mesh federation immediately. However, it will not be clear what is the difference with the single multi-cluster meshes. We can put the comparison at the end or we can add it later. Let's see what is the opinion of other people and consider the alternatives.

Reply:
I suggest you rework this into a series of blog posts, where each addresses a different use case of Istio topology. The first post can just be the concepts and identifying the use cases you'll tackle in subsequent blogs.